src/System.Security.Cryptography.Cng/src/System.Security.Cryptography.Cng.csproj

```diff
@@ -19,6 +20,7 @@
     <GeneratePlatformNotSupportedAssembly Condition="'$(TargetsUnix)' == 'true'">true</GeneratePlatformNotSupportedAssembly>
   </PropertyGroup>
   <PropertyGroup Condition="'$(TargetGroup)' == 'netcore50'">
+    <AssemblyVersion>4.1.0.0</AssemblyVersion>
     <PackageTargetFramework>netstandard1.4</PackageTargetFramework>
```

Comment:
Hmm, I would have expected it to use the default api list which is more permissive than NETCore50.  Regardless I added the 'SupportsUWP' property to address the scenario of a netstandard lib that has pinvokes that need to pass WACK.  The idea is you use [netstandard1.4](https://github.com/dotnet/corefx/blob/master/dir.props#L386) which is supported by the infra and set SupportsUWP=true.  Anyhow, I'm ok with this as is.  It's just a cleanup that doesn't impact packaging.